resources/Makefile

```diff
@@ -0,0 +1,17 @@
+run-silent:
+	docker-compose up -d
+
+run:
+	docker-compose up
+
+# run without the docker-compose.override.yml
+run-core:
+	docker-compose -f docker-compose.yml up
+
+# make local valid certs
+mkcert:
+	cd resources/ssl
+	mkert georchestra.mydomain.org
```

Comment:
mkcert ?

Reply:
you should call the target "cert", so one has to run `make cert`, which is way cooler ;-)

Reply:
https://github.com/FiloSottile/mkcert works on Windows, Linux, Mac. 
It does not exempt us to provide some certs, but since HSTS is activated by default by traefik, having a method to easily regenerate certs seems a + to me.

Reply:
:rofl: I know mkcert, thanks ;-)
What I meant is there is a typo in your Makefile : /mkert/mkcert/

Reply:
:facepalm: sorry...